spec/lib/scoreboard_renderer_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+require 'rails_helper'
+require 'scoreboard_renderer'
+
+RSpec.describe ScoreboardRenderer do
+  describe '.render' do
+    let(:participant) { create :participant }
+    let(:maneuver) { create :maneuver }
+    let(:output) { Nokogiri::HTML(described_class.render) }
+    let(:css) { ->(*args) { output.css(*args) } }
+    let(:sb) { 'table.scoreboard tbody' }
+    before do
+      create :maneuver_participant, maneuver: maneuver, participant: participant
+    end
+
+    it 'is an HTML document' do
+      expect(output > 'html').to be_present
+    end
+
+    it 'has non-debug stylesheets' do
+      stylesheets = css['head link[rel="stylesheet"]'].map { |s| s['href'] }
+      expect(stylesheets).not_to include(match('debug'))
+    end
+
+    it 'has non-debug javascript' do
+      js = css['script[src]'].map { |s| s['src'] }
+      expect(js).not_to include(match('debug'))
+    end
+
+    it 'has no navigation buttons' do
+      expect(css['nav #mainnav a']).to be_empty
```

Comment:
Will this pass if there's no CSS that even matches `nav #mainnav a` or will it throw up?

Reply:
If there were CSS that matched `nav #mainnav a`, then it wouldn't be empty.

Reply:
What I'm getting at is that what if I come along and decide that i'm going to change `#mainnav` to `#somethingelse` or we do a bootstrap update and `nav` isn't a thing?  This test would erroneously pass, right? I suppose it's not the biggest deal.

Reply:
There's another test for the existence of an `h1` in the nav. I can leverage that a bit to help guard against that, I think.